src/utils/utils.ts

```diff
@@ -1 +1,74 @@
-// Write utilities here
+import { Announcement } from "@/models/Announcement";
+
+/**
+ * Filters out announcements whose startDate is in the past.
+ *
+ * @param announcements - An array of Announcement objects.
```

Comment:
Since we are using a type-safe language, your docstrings should describe the parameter is used for instead of what the type is. For example, `The announcements to filter.`.